SiddhiBenchmark/LengthWindowBenchmark/src/main/java/Main.java

```diff
@@ -0,0 +1,136 @@
+import java.io.File;
```

Comment:
Lets have meaningful file name than Main.Java

Reply:
Changed the class name to Benchmark